apps/web/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "Node16",
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This was necessary to enable using the "new" `exports` field in `package.json` (in `novui/package.json`). This permits imports like `@novu/novui/css` which actually points to `./styled-system/css` instead of `dist`‏